nomad/local/grapl-local-infra.nomad

```diff
@@ -380,6 +553,10 @@ job "grapl-local-infra" {
               }
             }
           }
+
+          resources {
+            memory = 500
+          }
```

Comment:
remove so we can see if the other postgres fix did the trick?

Reply:
done, still seeing issues that appear to be caused by lack of postgres memory. In particular, seeing this repeatedly in the `plugin-work-queue-db` error logs: 
```
2022-08-08 17:49:46.642 UTC [1] LOG:  server process (PID 111) was terminated by signal 9: Killed
```